packages/react/src/themes/base/components/Text/textVariables.ts

```diff
@@ -1,8 +1,5 @@
-import { ColorValues } from '../../../types'
-import { mapColorsToScheme } from '../../../../lib'
-
 export interface TextVariables {
-  colors: ColorValues<string>
+  colorScheme: any
```

Comment:
type?

Reply:
Thanks for noticing, will have to take a look on how this will reflect on the base theme